S/solar/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "solar"
```

Comment:
Last change: the project seems to be called SOLAR
```suggestion
name = "SOLAR"
```
Need also to rename the directory accordingly